UefiPayloadPkg/Library/FdtParserLib/FdtParserLib.c

```diff
@@ -282,6 +290,15 @@ ParseReservedMemory (
           PlatformAcpiTable->Header.Revision = UNIVERSAL_PAYLOAD_ACPI_TABLE_REVISION;
           PlatformAcpiTable->Header.Length   = sizeof (UNIVERSAL_PAYLOAD_ACPI_TABLE);
         }
+      } else if (AsciiStrnCmp (TempStr, "smbios", AsciiStrLen ("smbios")) == 0) {
+        DEBUG ((DEBUG_INFO, " build smbios, NumberOfBytes:%x", NumberOfBytes));
+        BuildMemoryAllocationHob (StartAddress, NumberOfBytes, EfiBootServicesData);

```

Comment:
our verification showed Line 295 caused ASSERT. I think we do not need this step because we have  buildGuidHob below.

ASSERT due to below requirement: https://github.com/tianocore/edk2/blob/master/MdePkg/Library/PeiHobLib/HobLib.c#L849-L852

Reply:
Or we could handle this from bootloader side which always creates smbios memory buffer following page alignment in both address and length.

I would recommend that we keep it as-is for now and if required we can send separate PR to update it later. What do you think?

Reply:
let's leave it as-is for now.